datasets/code_search_net/README.md

```diff
@@ -0,0 +1,194 @@
+---
+annotations_creators:
+- no-annotation
+language_creators:
+- machine-generated
+languages: ["go", "php", "python", "java", "javascript", "ruby"]
```

Comment:
haha these languages are not part of the tagging taxonomy afaik ^^
Do you have any idea how to handle that @yjernite ?

Reply:
Thanks for the review @lhoestq ! I've reduced the dummy folder size to 48kB, would that do the trick ?